syftr/__init__.py

```diff
@@ -1,3 +1,8 @@
-from importlib.metadata import version as _version
+try:
+    from importlib.metadata import version
 
-__version__ = _version("syftr")
+    __version__ = version("syftr")
+except Exception:
+    import os
+
+    __version__ = os.getenv("SYFTR_VERSION", "0.0.0")
```

Comment:
I am confused: now we need to maintain version as an env var in addition to dynamic versioning?

Reply:
On the cluster, there is no package and no git repo and no version number. That's the root cause of all of this.

Reply:
> On the cluster, there is no package and no git repo and no version number. That's the root cause of all of this.

but there should be ray runtime env with pip attribute that should contain all dependencies?